README.rst

```diff
@@ -294,15 +294,17 @@ Starts listening structure changes and passes the changes to provided callback f
 
         node.subscribe_to_structure_changes(on_change)
 
-node.subscribe_to_value_changes(callback)
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+node.subscribe_to_value_changes(callback, fs=5)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
 Starts listening value changes and passes the changes to provided callback function
 
 - Arguments
 
     callback - Function(value, timestamp)
 
+    fs - maximum frequency the value changes are received. Defaults to 5 hz.
```

Comment:
Maybe say: "The requested frequency of value packet delivery to client. Defaults to 5 hz."

As it is not actually the maximum in case there is data to fill max packet size faster, they will arrive faster
and its not really about value change rate  - that is the sampling_frequencey.